proposals/Exceptions.md

```diff
@@ -444,8 +159,9 @@ Try blocks, like control-flow blocks, have a _block type_. The block type of a
 try block defines the values yielded by the evaluation the try block when either
 no exception is thrown, or the exception is successfully caught by the catch
 block.
+
 Try blocks, like control-flow blocks, have a _block type_. The block type of a
```

Comment:
This is the same as the previous paragraph, aside from the last sentence

Reply:
OMG... What's wrong with me. Thanks.